### PR TITLE
feat(console): color picker

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add component: color picker.

## Linear Issue Reference

LOG-1729

## Testing

image

image

### CODE REVIEW

**Review by simeng-li — APPROVED** [review]
LGTM

**Comment by demonzoo** [comment]
Build failed. Please fix

File: packages/console/src/pages/SignInExperience/components/ExperienceForm.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import ColorPicker from '@/components/ColorPicker';
+import FormField from '@/components/FormField';
+import RadioGroup, { Radio } from '@/components/RadioGroup';
+import TextInput from '@/components/TextInput';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  register: UseFormRegister<SignInExperience>;
+  control: Control<SignInExperience>;
```
Comment (demonzoo) [inline packages/console/src/pages/SignInExperience/components/ExperienceForm.tsx:15]: `useFormContext`

File: packages/console/src/components/ColorPicker/index.module.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  font: var(--font-body-medium);
+  position: relative;
+
+  &:focus-within {
+    border-color: var(--color-primary);
+    outline-color: var(--color-primary-80);
+  }
+
+  input {
+    position: absolute;
```
Comment (demonzoo) [inline packages/console/src/components/ColorPicker/index.module.scss:20]: why `absolute`?

File: packages/console/src/components/ColorPicker/index.module.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
+@use '@/scss/underscore' as _;
+
+.container {
+  border-radius: 6px;
+  border: 1px solid var(--color-component-border);
+  outline: 2px solid transparent;
```
Comment (demonzoo) [inline packages/console/src/components/ColorPicker/index.module.scss:6]: outline should be `3px`

File: packages/console/src/components/ColorPicker/index.module.scss [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  transition-property: outline, border;
+  transition-timing-function: ease-in-out;
+  transition-duration: 0.2s;
+  padding: _.unit(2) _.unit(3);
+  font: var(--font-body-medium);
+  position: relative;
+
+  &:focus-within {
+    border-color: var(--color-primary);
+    outline-color: var(--color-primary-80);
```
Comment (demonzoo) [inline packages/console/src/components/ColorPicker/index.module.scss:16]: outline-color should be `--color-focused-variant`

File: packages/console/src/components/ColorPicker/index.module.scss [hunk 5]
```diff
@@ -0,0 +1,40 @@
+@use '@/scss/underscore' as _;
+
+.container {
+  border-radius: 6px;
+  border: 1px solid var(--color-component-border);
```
Comment (demonzoo) [inline packages/console/src/components/ColorPicker/index.module.scss:5]: --color-border

File: packages/console/src/pages/SignInExperience/components/ExperienceForm.tsx (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/pages/SignInExperience/components/ExperienceForm.tsx]: rebased

File: packages/console/src/components/ColorPicker/index.module.scss (hunk 2 above)
Comment (wangsijie) [reply packages/console/src/components/ColorPicker/index.module.scss:20]: To make it covered by a div, by using this way we can change the apperence of the native color picker icon.

File: packages/console/src/components/ColorPicker/index.module.scss (hunk 5 above)
Comment (wangsijie) [reply packages/console/src/components/ColorPicker/index.module.scss]: updated

File: packages/console/src/components/ColorPicker/index.module.scss (hunk 4 above)
Comment (wangsijie) [reply packages/console/src/components/ColorPicker/index.module.scss]: updated

File: packages/console/src/components/ColorPicker/index.module.scss (hunk 3 above)
Comment (wangsijie) [reply packages/console/src/components/ColorPicker/index.module.scss]: updated

File: packages/console/src/components/ColorPicker/index.module.scss (hunk 2 above)
Comment (demonzoo) [reply packages/console/src/components/ColorPicker/index.module.scss:20]: Actually we can customize the native color input styles. Check this out:
https://stackoverflow.com/a/65021768/3431443

Comment (wangsijie) [reply packages/console/src/components/ColorPicker/index.module.scss:20]: Created LOG-2199

**Pull request merged**